views/products/show.slim

```diff
@@ -0,0 +1,9 @@
+h3 Add product to basket
+div
+  form action="/basket" method="POST" class="form-inline"
+    input type="hidden" name="product_id" value="#{product.id}"
+    div.form-group
+      label claas="control-label" style="margin-right: 10px; font-size: 17px;" Quantity:
```

Comment:
putting css (style) inline (inside html view) is very bad idea. Define new class and put it inside css file (as Maciek shows you all)